spec/models/course/discussion/post_spec.rb

```diff
@@ -41,9 +41,8 @@
         expect(root_post.first).to eq(graph[:root])
 
         root_children = root_post.second
-        expect(root_children).to contain_exactly([graph[:a], [
-                                                   [graph[:b], []]]],
-                                                 [graph[:c], []])
+        expect(root_children).
+          to contain_exactly([graph[:a], [[graph[:b], []]]], [graph[:c], []])
```

Comment:
This is a little hard to parse - not sure if we want to represent this visually using either some indentation here, or we can do it using a variable or comments, or using `Array.new(...)`. Let me see if I can come up with something more readable.